src/main/java/com/sh/threesentences/auth/dto/LoginDto.java

```diff
@@ -0,0 +1,16 @@
+package com.sh.threesentences.auth.dto;
+
+import lombok.Getter;
+
+@Getter
+public class LoginDto {
+
+    String email;
+
+    String password;
```

Comment:
```suggestion
    private final String email;

    private final String password;
```

중요한건 아니지만 잊지 않고 붙여주시면 좋을 것 같아요~